javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -59,45 +61,271 @@ const playerTwoMarker = "O";
 
 // example two
 const playerOne = {
-  name: "tim",
-  marker: "X"
+  name: "tim",
+  marker: "X"
 };
 
 const playerTwo = {
-  name: "jenn",
-  marker: "O"
+  name: "jenn",
+  marker: "O"
 };
 ```
 
-At first glance, the first doesn't seem so bad.. and it actually takes fewer lines to write than the example using objects, but the benefits of the second approach are huge! Let me demonstrate:
+At first glance, the first doesn’t seem so bad.. and it actually takes fewer lines to write than the example using objects, but the benefits of the second approach are huge! Grouping related data together into objects allows you to pass the data around easily. Let me demonstrate:
```

Comment:
```suggestion
At first glance, the first doesn't seem so bad.. and it actually takes fewer lines to write than the example using objects, but the benefits of the second approach are huge! Grouping related data together into objects allows you to pass the data around easily. Let me demonstrate:
```
Somehow the apostrophes in this PR were adjusted to a different character. You'll want to fix all these to have them be how they were originally.

Reply:
This and the javascript/`js` code block changes were probably due to me copying/pasting from a markdown preview tool. I probably should have fixed the `js` code tag when I noticed it in the dif, but I did not. Didn't notice the apostrophes. Consider them fixed.